priv/templates/nodetool

```diff
@@ -50,6 +50,17 @@ main(Args) ->
                         {badrpc, Reason} ->
                             io:format("RPC to ~p failed: ~p~n", [TargetNode, Reason]),
                             halt(1);
+                        rpc_ok ->
+                            %% code doesn't want any output
+                            ok;
+                        {rpc_error, Code} ->
```

Comment:
can you put a guard on the type there so that if we don't respect the `Code` expected by `halt/1` we output the result rather than crashing silently with a bad code?

Reply:
sure thing.